naming/src/main/java/com/alibaba/nacos/naming/consistency/persistent/impl/BasePersistentServiceProcessor.java

```diff
@@ -121,11 +121,10 @@ public BasePersistentServiceProcessor(final ClusterVersionJudgement judgement) t
                 throw new NacosRuntimeException(ex.getErrCode(), ex.getErrMsg());
             }
         });
-        afterConstruct();
     }
     
     @SuppressWarnings("unchecked")
-    protected void afterConstruct() {
+    public void afterConstruct() {
         NotifyCenter.registerToPublisher(ValueChangeEvent.class, 16384);
```

Comment:
Why is there a magic number here？

Reply:
This is the queue length for cached events